environment/deployments/panda/env/dev-gke.tfvars

```diff
@@ -61,18 +62,37 @@ node_pools_non_preempt_0 = [
     local_ssd_count    = 0
     auto_repair        = true
     auto_upgrade       = true
-    preemptible        = true
+    preemptible        = false
     image_type         = "cos_containerd"
     enable_secure_boot = true
     disk_size_gb       = "200"
     disk_type          = "pd-standard"
     autoscaling        = true
-    node_count         = 0
+    node_count         = 1
     min_count = 0
-    max_count = 1000
+    max_count = 10
   }
  ]
 
+node_pools_merge_0 = [
+  {
+    name               = "node_pools_merge_0"
+    machine_type       = "n2-standard-4"
+    node_locations     = "us-central1-c"
+    local_ssd_count    = 0
+    auto_repair        = true
+    auto_upgrade       = true
+    preemptible        = true
```

Comment:
Do you mean to have this one preemptible or non-preemtible?

Reply:
Yes, since Tim reduces the merge job execution time. If needed this property could be updated.